### PR TITLE
CreatingVendorPage UI

### PR DESCRIPTION
**Pull request by czhou578**
creating the UI for the creatingVendorPage.

### CODE REVIEW

**Comment by czhou578** [comment]
This is kind of petty, but the padding is a bit off on the label here, and I don't know how to change it. Does it bother you?

image

File: app/src/components/UI/Pages/CreateVendorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        <Input
+          type="file"
+          label="Business Logo"
+          className={styles.input}
+          size="small"
+          fluid
+        />
+
+        <Form.Field
```
Comment (MakotoE) [inline app/src/components/UI/Pages/CreateVendorPage.tsx:66]: Not all vendors are food trucks.

File: app/src/components/UI/Pages/CreateVendorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          />
+          <Form.Field
+            control={Input}
+            label="Website URL"
+            placeholder="Website URL"
+            width={8}
+          />
+        </Form.Group>
+
+        <Input
```
Comment (MakotoE) [inline app/src/components/UI/Pages/CreateVendorPage.tsx:58]: The off-centered label is annoying, but I notice that all other fields have labels right above the input. So this label should also follow the same convention.

**Review by MakotoE — CHANGES_REQUESTED** [review]

**Comment by czhou578** [comment]
Looks better.
image

**Review by MakotoE — APPROVED** [review]

**Pull request merged**